zoltar-models/src/main/java/com/spotify/zoltar/ModelLoaders.java

```diff
@@ -0,0 +1,102 @@
+/*-
+ * -\-\-
+ * zoltar-models
+ * --
+ * Copyright (C) 2016 - 2018 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.zoltar;
+
+import com.spotify.zoltar.tf.TensorFlowGraphLoader;
+import com.spotify.zoltar.tf.TensorFlowGraphModel;
+import com.spotify.zoltar.tf.TensorFlowLoader;
+import com.spotify.zoltar.tf.TensorFlowModel;
+import com.spotify.zoltar.xgboost.XGBoostLoader;
+import javax.annotation.Nullable;
+import org.tensorflow.Graph;
+import org.tensorflow.framework.ConfigProto;
+
+/**
+ * This class consists exclusively of static methods that return Model Loaders.
+ *
+ * <p>This is the public entry point for get get a Model.</p>
+ */
+public final class ModelLoaders {
```

Comment:
This I guess deprecates `Models` right, could you make make it private if still used internally or remove if obsolete?

Reply:
👍 removing it.